content/geoip/docs/databases/enterprise.mdx

```diff
@@ -203,9 +203,9 @@ name would be "GeoIP2-City-Locations-en.csv".
 
 ### Example Files
 
-This zip file contains all examples of all Enterprise CSV databases, limited
-to a single IP address of 67.248.44.118 -
-[GeoIP2-Enterprise-CSV_Example](/static/GeoIP2-Enterprise-CSV_Example.zip)
+This zip file contains examples of all GeoIP2 Enterprise CSV databases:
```

Comment:
The wording on this seems better than it was, but it is still confusing, in particular "all ... databases". Although I would probably shy away from "database" for the CSVs generally, if I were to use it, I would use it for the collection of CSV files in the zip. I think maybe replacing it with "files" would be better here.

This also applies to the ISP one.